src/main/java/uk/gov/hmcts/reform/wataskmonitor/services/CamundaService.java

```diff
@@ -0,0 +1,62 @@
+package uk.gov.hmcts.reform.wataskmonitor.services;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.core.io.ClassPathResource;
+import org.springframework.stereotype.Component;
+import org.springframework.util.FileCopyUtils;
+import uk.gov.hmcts.reform.wataskmonitor.clients.CamundaClient;
+import uk.gov.hmcts.reform.wataskmonitor.exceptions.CamundaRequestFailure;
+import uk.gov.hmcts.reform.wataskmonitor.models.CamundaTask;
+
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.nio.charset.StandardCharsets;
+import java.time.ZonedDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.List;
+
+@Component
+@Slf4j
+public class CamundaService {
+
+    public static final String QUERY_PARAMETERS_JSON = "camundaquery/camunda-query-parameters.json";
```

Comment:
Is this temporary?

Reply:
No, not really. The camunda query to retrieve unconfigured tasks is always the same. So I have just put it in that JSON and then load the JSON to run the query. It saves us creating models etc. that we don't really need to manipulate

Reply:
Ok for now but the Task Monitor has a wider purpose than this so in future we will be sending queries with different parameters.  So I think we'll have to create models etc then.